sdk/confidentialledger/azure-confidentialledger/azure/confidentialledger/receiptverification/models.py

```diff
@@ -0,0 +1,188 @@
+# ------------------------------------
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+# ------------------------------------
+"""Models for receipt verification."""
+
+from typing import List, Optional
+from azure.confidentialledger.receiptverification._serialization import (
+    Model,
+)
+
+
+class ProofElement(Model):
```

Comment:
Why are these inheriting from Model? We should avoid that base class.
Additionally - why is this module public? These are already exposed via __init__.py

Reply:
They are inheriting from `Model` so that we can simplify the conversion from / to JSON objects and do validation of the types and fields. Why should the `Model` class be avoided?

I will make this file private, thanks!

Reply:
File made private https://github.com/Azure/azure-sdk-for-python/pull/27265/commits/28f7642c76da39c07e66c9cfae35a7dff46807f0

Reply:
See earlier comment. The `Model` class is only intended for getting data to and from the network/wire format.

Reply:
Removed the serialization file and these classes are not inheriting from Model anymore.